package.json

```diff
@@ -1,34 +1,67 @@
 {
-  "name": "@oasisprotocol/deoxysii",
-  "version": "0.0.5",
-  "description": "Deoxys-II-256-128",
-  "main": "./deoxysii",
-  "types": "./deoxysii.d.ts",
-  "scripts": {
-    "build": "tsc",
-    "bench": "node ./bench/bench.js",
-    "test": "mocha",
-    "lint": "eslint *.js ./test ./bench"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/oasisprotocol/deoxysii-js.git"
-  },
-  "license": "MIT",
-  "author": "Oasis Protocol Foundation <info@oasisprotocol.org>",
-  "devDependencies": {
-    "@types/chai": "^4.3.4",
-    "@types/mocha": "^10.0.1",
-    "@types/node": "^18.15.11",
-    "@types/uint32": "^0.2.0",
-    "benchmark": "^2.1.4",
-    "chai": "*",
-    "eslint": "^5.14.0",
-    "mocha": "^6.1.4",
-    "typescript": "^5.0.4"
-  },
-  "dependencies": {
-    "bsaes": "0.0.2",
-    "uint32": "^0.2.1"
-  }
+	"name": "@oasisprotocol/deoxysii",
+	"version": "0.0.6",
+	"description": "Deoxys-II-256-128",
+	"sideEffects": false,
+	"files": [
+		"dist",
+		"!dist/*.tsbuildinfo",
+		"LICENSE",
+		"src",
+		"!src/*.test.ts",
+		"!src/*.bench.ts"
+	],
+	"main": "./dist/index.js",
+	"module": "./dist/index.mjs",
+	"types": "./dist/index.d.ts",
+	"type": "commonjs",
+	"exports": {
+		".": {
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.js"
+			},
+			"import": {
+				"types": "./dist/index.d.mts",
+				"default": "./dist/index.mjs"
+			}
+		}
+	},
+	"scripts": {
+		"bench": "pnpm vitest bench src/*.bench.ts",
+		"test": "vitest",
+		"coverage": "vitest run --coverage --coverage.exclude=scripts --coverage.exclude='src/*.bench.ts' --coverage.exclude='examples'",
+		"lint": "biome check .",
+		"format": "biome format --write .",
+		"clean": "rm -rf dist coverage",
+		"distclean": "pnpm run clean && rm -rf node_modules *.tgz",
+		"build": "pnpm tsup src/index.ts --format cjs,esm --dts --clean",
+		"prepublishOnly": "npm run build",
+		"debug-imports-cjs": "NODE_DEBUG=module,esm bash -c 'pnpm node --experimental-global-webcrypto examples/check.cjs 2> >(grep deoxysii)'",
```

Comment:
I don't understand this. Is it just
`NODE_DEBUG=module,esm node --experimental-global-webcrypto examples/check.cjs | grep deoxysii`
minus the errors?

Reply:
It greps stderr for logs from `module` or `esm` to show only ones related to deoxysii.

Essentially this double checks that check.cjs loads the .js file via the normal module loader, and check.mjs loads the .mjs file via ESM

Reply:
```

> @oasisprotocol/deoxysii@0.0.6 debug-imports /home/runner/work/deoxysii-js/deoxysii-js
> pnpm run debug-imports-cjs && pnpm run debug-imports-esm


> @oasisprotocol/deoxysii@0.0.6 debug-imports-cjs /home/runner/work/deoxysii-js/deoxysii-js
> NODE_DEBUG=module,esm bash -c 'pnpm node --experimental-global-webcrypto examples/check.cjs 2> >(grep deoxysii)'

MODULE 2161: Module._load REQUEST /home/runner/work/deoxysii-js/deoxysii-js/.pnpmfile.cjs parent: /home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/dist/pnpm.cjs
MODULE 2161: looking for "/home/runner/work/deoxysii-js/deoxysii-js/.pnpmfile.cjs" in ["/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/dist/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules","/home/runner/node_modules","/home/node_modules","/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/.node_modules","/home/runner/.node_libraries","/opt/hostedtoolcache/node/18.20.2/x64/lib/node"]
MODULE 2179: looking for "/home/runner/work/deoxysii-js/deoxysii-js/examples/check.cjs" in ["/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/.node_modules","/home/runner/.node_libraries","/opt/hostedtoolcache/node/18.20.2/x64/lib/node"]
MODULE 2179: load "/home/runner/work/deoxysii-js/deoxysii-js/examples/check.cjs" for module "."
MODULE 2179: Module._load REQUEST @oasisprotocol/deoxysii parent: .
MODULE 2179: looking for "@oasisprotocol/deoxysii" in ["/home/runner/work/deoxysii-js/deoxysii-js/examples/node_modules","/home/runner/work/deoxysii-js/deoxysii-js/node_modules","/home/runner/work/deoxysii-js/node_modules","/home/runner/work/node_modules","/home/runner/node_modules","/home/node_modules","/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/nocjs [class AEAD]
Encrypted: Uint8Array(27) [
  233,  24, 247,  21,  97, 209, 159,
  253,   5, 151, 186,  13, 204,  42,
  227, 216, 204, 121,  32, 176, 232,
   24,  16, 216, 130, 102, 206
]
Decrypted: Hello World
de_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/.node_modules","/home/runner/.node_libraries","/opt/hostedtoolcache/node/18.20.2/x64/lib/node"]
MODULE 2179: load "/home/runner/work/deoxysii-js/deoxysii-js/dist/index.js" for module "/home/runner/work/deoxysii-js/deoxysii-js/dist/index.js"

> @oasisprotocol/deoxysii@0.0.6 debug-imports-esm /home/runner/work/deoxysii-js/deoxysii-js
> NODE_DEBUG=module,esm bash -c 'pnpm node --experimental-global-webcrypto examples/check.mjs 2> >(grep deoxysii)'

star [class AEAD]
default [class AEAD]
AEAD [class AEAD]
Encrypted: Uint8Array(27) [
  175, 189, 113,  76, 235, 191, 199,
   63, 217, 208,  56, 109,  32, 214,
  207, 137, 242,   9,   9, 141,   6,
  220, 222,  46, 100, 147, 174
]
Decrypted: Hello World
MODULE 2204: Module._load REQUEST /home/runner/work/deoxysii-js/deoxysii-js/.pnpmfile.cjs parent: /home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/dist/pnpm.cjs
MODULE 2204: looking for "/home/runner/work/deoxysii-js/deoxysii-js/.pnpmfile.cjs" in ["/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/dist/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules","/home/runner/node_modules","/home/node_modules","/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/bin/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules/pnpm/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/pnpm@8.15.8/node_modules","/home/runner/setup-pnpm/node_modules/.pnpm/node_modules","/home/runner/.node_modules","/home/runner/.node_libraries","/opt/hostedtoolcache/node/18.20.2/x64/lib/node"]
ESM 2222: Storing file:///home/runner/work/deoxysii-js/deoxysii-js/examples/check.mjs (implicit type) in ModuleLoadMap
ESM 2222: Translating StandardModule file:///home/runner/work/deoxysii-js/deoxysii-js/examples/check.mjs
ESM 2222: Storing file:///home/runner/work/deoxysii-js/deoxysii-js/dist/index.mjs (implicit type) in ModuleLoadMap
ESM 2222: Translating StandardModule file:///home/runner/work/deoxysii-js/deoxysii-js/dist/index.mjs
```